go/cmd/vtctldclient/command/vreplication/lookupvindex/lookupvindex.go

```diff
@@ -158,7 +179,7 @@ var (
 	// externalize makes a LookupVindexExternalize call to a vtctld.
 	externalize = &cobra.Command{
 		Use:                   "externalize",
-		Short:                 "Externalize the Lookup Vindex. If the Vindex has an owner the VReplication workflow will also be deleted.",
+		Short:                 "Externalize the Lookup Vindex. If the Vindex has an owner the VReplication workflow will also be stopped/deleted.",
```

Comment:
This text is a bit off IMO. If we're not able to set the owner (the backing table) as *part of the externalize* then it's an error. No? Unless I'm missing something, then I think this is better:
```
"Externalize the Lookup Vindex and stop (or optionally delete) the workflow."
```

Reply:
Actually, we don't return an error if the vindex is unowned while externalizing, instead we just remove the `write_only` param from the vindex. This is the old behaviour that we follow in `LookupVindexExternalize`. Do you think we should change this behaviour (we can return error for unowned vindexes)?

Reply:
OK, fine to leave it as is then. Been a while since I was in this code so had forgotten. Thanks!